biological-and-pharmaceutical-bulletin.csl

```diff
@@ -0,0 +1,249 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Biological and Pharmaceutical Bulletin</title>
+    <title-short>Biol. Pharm. Bull.</title-short>
+    <id>http://www.zotero.org/styles/biological-and-pharmaceutical-bulletin</id>
+    <link href="http://www.zotero.org/styles/biological-and-pharmaceutical-bulletin" rel="self"/>
+    <link href="http://bpb.pharm.or.jp/document/inst_e.pdf" rel="documentation"/>
+    <author>
+      <name>Shoji Takahashi</name>
+      <email>s.takahashi@elsevier.com</email>
+      <uri>http://www.mendeley.com/profiles/shoji-takahashi3/</uri>
+    </author>
+    <category citation-format="numeric"/>
+    <issn>0918-6158</issn>
+    <eissn>1347-5215</eissn>
+    <updated>2015-12-12T11:03:34+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <style-options punctuation-in-quote="false"/>
+  </locale>
+  <macro name="author">
+    <names variable="author">
+      <name initialize-with="" name-as-sort-order="all" sort-separator=" "/>
+    </names>
+  </macro>
+  <macro name="container-title">
+    <choose>
+      <if type="article-journal" match="any">
+        <text variable="container-title" form="short" font-style="italic"/>
+      </if>
+      <else-if type="chapter" match="any">
+        <text variable="container-title" font-style="italic"/>
+      </else-if>
+      <else>
+        <text variable="container-title" font-style="normal"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="edition">
+    <number suffix=" " variable="edition" form="ordinal"/>
+    <label plural="never" variable="edition" form="short"/>
+  </macro>
+  <macro name="event">
+    <group delimiter=", ">
+      <text variable="event-place"/>
+      <group>
+        <text value="on " suffix=" "/>
+        <date variable="event-date">
+          <date-part name="day" range-delimiter="" suffix=" "/>
+          <date-part name="month" range-delimiter="" suffix=", "/>
+          <date-part name="year" range-delimiter=""/>
```

Comment:
this won't really do anything in Mendeley/Zotero at the moment, but remove the range-delimiters here. Otherwise you'd get ranges without anything between them (MayJune, 2003) etc.